consensus/byzantine_test.go

```diff
@@ -113,7 +113,7 @@ func TestByzantinePrevoteEquivocation(t *testing.T) {
 	bzReactor := rts.reactors[bzNodeID]
 
 	// alter prevote so that the byzantine node double votes when height is 2
-	bzNodeState.doPrevote = func(height int64, round int32) {
+	bzNodeState.doPrevote = func(height int64, round int32, mustVoteNil bool) {
```

Comment:
since this isn't used
```suggestion
	bzNodeState.doPrevote = func(height int64, round int32, _ bool) {
```